ShippoTesting/ManifestTest.cs

```diff
@@ -1,47 +1,84 @@
-﻿using NUnit.Framework;
+﻿﻿using NUnit.Framework;
 using System;
 using System.Collections;
+using System.Collections.Generic;
+
 using Shippo;
 
-namespace ShippoTesting {
-    [TestFixture ()]
-    public class ManifestTest : ShippoTest {
 
-        [Test ()]
-        public void TestInvalidCreate ()
+namespace ShippoTesting
+{
+    [TestFixture]
+    public class ManifestTest : ShippoTest
+    {
+
+        [Test]
+        public void TestInvalidCreate()
         {
-			Assert.That (() => ManifestTest.getDefaultObject (),Throws.TypeOf<ShippoException> ());
+			Assert.That(() => ManifestTest.getDefaultObject(), Throws.TypeOf<ShippoException>());
         }
 
-        [Test ()]
-        public void testValidRetrieve ()
+        [Test]
+        public void testValidRetrieve()
         {
-            Manifest testObject = ManifestTest.getDefaultObject ();
+            Manifest testObject = ManifestTest.getDefaultObject();
             Manifest retrievedObject;
 
-            retrievedObject = apiResource.RetrieveManifest ((string) testObject.ObjectId);
-            Assert.AreEqual (testObject.ObjectId, retrievedObject.ObjectId);
+            retrievedObject = apiResource.RetrieveManifest((string) testObject.ObjectId);
+            Assert.AreEqual(testObject.ObjectId, retrievedObject.ObjectId);
         }
 
-        [Test ()]
-        public void testListAll ()
+        [Test]
+        public void testListAll()
         {
-            Hashtable parameters = new Hashtable ();
-            parameters.Add ("results", "1");
-            parameters.Add ("page", "1");
+            Hashtable parameters = new Hashtable();
+            parameters.Add("results", "1");
+            parameters.Add("page", "1");
 
-            var Manifests = apiResource.AllManifests (parameters);
-            Assert.AreEqual (0, Manifests.Data.Count);
+            var Manifests = apiResource.AllManifests(parameters);
+            Assert.AreEqual(1, Manifests.Data.Count);
```

Comment:
why are we now expecting one manifest? It doesn't seem any other code changed to expect this, unless of course our API itself changed/ seed database